composer.json

```diff
@@ -22,10 +22,10 @@
         "phpunit/phpunit": "^4.7"
     },
     "autoload": {
-        "psr-0": { "WindowsAzure\\": "WindowsAzure/" }
+        "psr-4": { "src\\": "WindowsAzure/" }
```

Comment:
Of course @yaqiyang, you're right. Making blind changes is quite annoying.
Updated. Thank you.